### ISSUE TITLE
Add badge to GitHub Pages

### ISSUE THREAD

**Issue by arboriculteurs** [dwyl/hits#60]
Is it possible to have the badge displayed not in the Readme file but in the index page of a website hosted on GitHub Pages ? Thanks a lot

**Comment by nelsonic** [comment]
@arboriculteurs good question is that a feature you would find _useful_? 🤔

**Comment by arboriculteurs** [comment]
Thanks a lot for your prompt response... 
Yes, I find it very useful. Because I learn to associations how to create free websites with Jekyll. And because of EU regulations, we want a web counter that works on Github pages, do not rely on external services with advertisements et uses no cookies. So no Google stats...
Your badge seems to be perfect if it could be put on another page than the readme.

**Comment by nelsonic** [comment]
@arboriculteurs have you _tried_ it...? 😉 (_it's a general web counter so it will work **anywhere**_)
(_also for GDPR compliance you need to be in **control** of the data that is being collected, so if a user of your service requests deletion of their personal data (_of which IP address counts_), you must be able to delete it. So using the Hits badge which pushes IP address and "page view" data to **my** database is probably not compliant from your perspective..._)
Given that all the code is Open Source you consider adapting it to run on your own App e.g. on Heroku Free tier. and thus maintain control of all the data and be able to run your own analysis.

**Comment by arboriculteurs** [comment]
Yes, of course, I did. I putted the markdown snippet on the footer of a Jekyll generated page, but nothing happened. 
I have not noticed that you collect the IP to... But an IP without any other personnal information should be considered compliant with GDPR.
Thanks for all the advices : I will try again.

**Comment by nelsonic** [comment]
@arboriculteurs unless you have a markdown parsing step in your Jekyll build it won't work like that.
You need to use HTML. 
Please note that Hits was not intended to be a "analytics as a [free] service" so definitely consider running your _own_ instance of Hits on Heroku or equivalent free hosting service.

**Comment by nelsonic** [comment]
This has worked for a `while`. Closing. ✅
